### PR TITLE
Adding new compare function to catch a feature having website address

### PR DESCRIPTION
**Pull request by chtnha**

### CODE REVIEW

**Comment by chtnha** [comment]
@batpad @amishas157 please review this compare function and merge. Thank you

File: comparators/added-website.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = addedWebsite;
+
+function addedWebsite(newVersion, oldVersion, callback) {
+  if (oldVersion && oldVersion.properties && newVersion && newVersion.properties && oldVersion.properties.website === newVersion.properties.website) {
+    callback(null, {});
+  } else if (newVersion && newVersion.properties && 'website' in newVersion.properties) {
+    callback(null, {'result:added-website': true});
+  } else if (oldVersion && oldVersion.properties && ('website' in oldVersion.properties) && newVersion && newVersion.properties && !('website' in newVersion.properties)) {
```
Comment (amishas157) [inline comparators/added-website.js]: @chtnha Not really sure if we need this condition. 🤔

File: tests/fixtures/added-website.json [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+        "geometry": {
+          "type": "Point",
+          "coordinates": [
+            76.85708999633789,
+            16.828531862816284
+          ]
+        }
+      },
+      "expectedResult": {
+        "result:added-website": true
```
Comment (amishas157) [inline tests/fixtures/added-website.json]: oldVersion has property called `website` but newVersion doesn't. That means the tag has been deleted . Therefore it should not be a suspicions for this reason.
@chtnha

File: comparators/added-website.js (hunk 1 above)
Comment (bkowshik) [inline comparators/added-website.js]: @amishas157 this is checking a the website existed in the `oldVersion` but does not in the `newVersion`. The usecase is somebody removing a website from a feature.

File: tests/fixtures/added-website.json (hunk 2 above)
Comment (bkowshik) [inline tests/fixtures/added-website.json]: @amishas157 We thought about this a little and for this iteration wanted to flag even deletions of `website`.

Comment (amishas157) [inline tests/fixtures/added-website.json]: @bkowshik Sounds fine then. PR looks good to me. Going ahead to merge it in that case.

**Pull request merged**